src/main.py

```diff
@@ -128,24 +128,22 @@ def do_activate(self):
         We raise the application"s main window, creating it if
         necessary.
         """
-        win = self.props.active_window
-        if not win:
-            win = GraphsWindow(application=self)
-        self.main_window = win
-        win.set_title(self.name)
+        self.main_window = self.props.active_window
+        if not self.main_window:
+            self.main_window = GraphsWindow(application=self)
+        self.main_window.set_title(self.name)
         if "(Development)" in self.name:
-            win.add_css_class("devel")
-        config = self.preferences.config
-        self.plot_settings = PlotSettings(config)
+            self.main_window.add_css_class("devel")
+        self.plot_settings = PlotSettings(self.preferences)
         pyplot.rcParams.update(
             file_io.parse_style(plot_styles.get_preferred_style(self)))
         self.canvas = Canvas(self)
-        win.toast_overlay.set_child(self.canvas)
-        win.redo_button.set_sensitive(False)
-        win.undo_button.set_sensitive(False)
+        self.main_window.toast_overlay.set_child(self.canvas)
+        self.main_window.redo_button.set_sensitive(False)
+        self.main_window.undo_button.set_sensitive(False)
         ui.enable_data_dependent_buttons(self)
         self.set_mode(None, None, InteractionMode.PAN)
-        win.present()
+        self.main_window.present()
```

Comment:
I didn't really notice before that we assign self.main_window, and then use win anyway. Giving us three different variables all refering to the same object in memory (`self.main_window`, `self.props.active_window` and `win`). 

I'm pretty sure that it's functionally identical, as they all point to the same object in memory, it's not exactly clean. So good change.